<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowLabels"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionUndoTest"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFactorNodes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowGrid"/>
   <addaction name="actionSnapGrid"/>
   <addaction name="separator"/>
   <addaction name="actionBranches"/>
   <addaction name="actionFans"/>
   <addaction name="separator"/>
   <addaction name="actionZoom"/>
   <addaction name="actionUnzoom"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="toolTip">
    <string>Enable/Disable Grid</string>
   </property>
  </action>
  <action name="actionSnapGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snap to Grid</string>
   </property>
   <property name="toolTip">
    <string>Snap Items to Grid</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
   <property name="toolTip">
    <string>Select Items</string>
   </property>
  </action>
  <action name="actionBranches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Branches</string>
   </property>
   <property name="toolTip">
    <string>Add Branches</string>
   </property>
  </action>
  <action name="actionFans">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fans</string>
   </property>
   <property name="toolTip">
    <string>Add Fans</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionUnzoom">
   <property name="text">
    <string>Unzoom</string>
   </property>
  </action>
  <action name="actionShowLabels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Labels</string>
   </property>
  </action>
  <action name="actionUndoTest">
   <property name="text">
    <string>Undo Test</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFactorNodes">
   <property name="text">
    <string>Factor Nodes...</string>
   </property>
   <property name="toolTip">
    <string>Multible nodes coordinates by the given factor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
